src/main/kotlin/no/nav/tilleggsstonader/sak/vilkår/stønadsperiode/StønadsperiodeValideringUtil.kt

```diff
@@ -10,18 +10,32 @@ import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.dto.Datoperiode
 import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.dto.VilkårperioderDto
 import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.dto.formattertPeriodeNorskFormat
 import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.dto.mergeSammenhengendeOppfylteVilkårperioder
+import java.time.LocalDate
 
 object StønadsperiodeValideringUtil {
 
+    /**
+     * Validering av stønadsperioder som kjøres når man endrer vilkårperiode trenger ikke å validere fødselsdatoet.
+     * Det er tilstrekkelig at det gjøres vid validering av stønadsperioder.
+     */
```

Comment:
Er denne kommentaren nødvendig? 
Støtter at det kun er nødvendig ved opprettelse

Reply:
/ er det egentlig nødvendig å ha to ulike funksjoner for dette bare for å slippe å gjøre den valideringen igjen?
Siden den er nullable vil det vell fortsatt fungere for tidligere behandlinger

Reply:
Tja litt for å dokumentere hvorfor jeg har null på fødselsdato. Var litt for å unngå å hente opp data fra grunnlagsdata når det ikke var noe særlig behov for det